turborepo-tests/examples/test_example.sh

```diff
@@ -0,0 +1,53 @@
+# Start by figuring out which example we're testing and its package manager
+example_path=$1
+package_manager=$2
+
+# If you need to do some debugging, you can set these to true
+debug=false
+force=false
```

Comment:
Not blocking at all, but might be nice to be able to control these via an env var.

Reply:
I spent more time than I care to admit trying to make an ergonomic way to handle that and it wasn't worth the complexity, in my opinion.